site/content/en/docs/tasks/setup_multikueue.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Setup a MultiKueue environment"
+date: 2024-02-26
+weight: 9
+description: >
+  Additional steps needed to setup the multikueue clusters.
```

Comment:
I don't think that helps, especially since the end goal for JobSet, for instance, is to not have different configurations .

From a Kueue's perspective I think it's covered.

Reply:
To have the same configuration for JobSet we need first support for managed by in kueue and in JobSet. I think the docs here should guide the admin step by step what to do given the current versions. We may mention in a note that we are working on simplifying things. Cc @mwielgus

Reply:
+1 on @mimowo 
We used to put set-by-setp introduction this `tasks` section.

Reply:
added